near-sdk-macros/res/near_blockchain.rs

```diff
@@ -105,6 +107,8 @@ pub mod near_blockchain {
             sys::log_utf16(len, ptr)
         }
 
+        unsafe fn abort(&self, msg_ptr: u32, filename_ptr: u32, line: u32, col: u32) {}
```

Comment:
Is there `sys::abort` ?

Reply:
Not sure what you mean.  This function is present in `near-vm-logic` so it should be mocked here as well and is used by errors in AS.